wordpress/wp-content/plugins/cds-base/classes/Modules/TrackLogins/TrackLogins.php

```diff
@@ -16,7 +16,12 @@ public function __construct()
     {
         global $wpdb;
         $this->wpdb      = $wpdb;
-        $this->tableName = $this->wpdb->prefix . 'userlogins';
+
+        if (property_exists($wpdb, 'base_prefix')) {
+            $this->tableName = $wpdb->base_prefix . 'userlogins';
+        } else {
+            $this->tableName = $this->wpdb->prefix . 'userlogins';
+        }
```

Comment:
Is there a chance base_prefix doesn't exist?

Also, should we add something to cleanup/remove the existing tables? (maybe they don't matter as we won't migrate them to prod and eventually we'll wipe staging)